src/edu/csus/ecs/pc2/shadow/RemoteEventFeedMonitor.java

```diff
@@ -25,7 +25,12 @@
 import edu.csus.ecs.pc2.core.model.ClientType.Type;
 import edu.csus.ecs.pc2.core.model.IFile;
 import edu.csus.ecs.pc2.core.model.RunUtilities;
+import edu.csus.ecs.pc2.ui.MessageManager;
+import edu.csus.ecs.pc2.ui.MessageRecord;
+import edu.csus.ecs.pc2.ui.MessageScope;
 import edu.csus.ecs.pc2.ui.ShadowCompareRunsPane;
+//import edu.csus.ecs.pc2.ui.ShadowControlPane;
```

Comment:
These should be removed.  The history is always available using GIT.  Leaving old code, just makes things confusing.

Reply:
removed, pushed fix